docs/README.Windows.md

```diff
@@ -261,3 +263,26 @@ UWP builds generate `msix`, `appxsym` and `cer` files, located inside directorie
 
 
 **[back to top](#table-of-contents)** | **[back to section top](#6-build-kodi-manually)**
+
+## 7. Build binary add-ons
+You can find a complete list of available binary add-ons **[here](https://github.com/xbmc/repo-binary-addons)**.
+
+### 7.1. In-tree building of binary add-ons
+Change to the 64bit build directory (**recommended**) or choose the appropriate architecture:
+```
+cd %userprofile%\kodi\tools\buildsteps\windows\x64
+```
+Run add-ons bootstrap:
+```
+bootstrap-addons.bat
+```
+Build specific add-on:
+```
+make-addons.bat package audioencoder.flac
+```
+Builded files are under the folder `kodi\addons\<add-on name>`.
+
+Clean-up binary add-ons:
+```
+make-addons.bat clean
+```
```

Comment:
I think some caveats should be mentioned to head off the inevitable questions, how do i build PR XYZ, how do i build branch X, how do i build from a fork.

The answer to all those currently is, you cant easily. At least not easily enough to put in a simplified readme doc.

Reply:
i add a more detailed description on chapter 7 about this, try take a look